lib/components/mobile/results-header.js

```diff
@@ -100,18 +100,13 @@ class ResultsHeader extends Component {
 
 const mapStateToProps = (state, ownProps) => {
   const activeSearch = getActiveSearch(state)
-  const {useRealtime} = state.otp
-  const response = !activeSearch
-    ? null
-    : useRealtime ? activeSearch.response : activeSearch.nonRealtimeResponse
+  const response = !activeSearch ? null : activeSearch.response
```

Comment:
Would this work?
```suggestion
  const response = activeSearch?.response
```

Reply:
Good call thanks! It's so easy to get lost while refactoring something down like this. Cleaned up in 23ef058a46835bd8ce5710176ca1671ffcf7a586